src/schema.py

```diff
@@ -40,6 +40,8 @@
     "linear_classification",
     "relu_2nn_regression",
     "decision_tree",
+    "kalman_filter",
+    "noisy_kalman_filter"
```

Comment:
Let's move this whole array (`TASK_LIST`) to `consts.py` and import it like we do `SEQ_MODELS`. It'll last longer there

Reply:
honestly yeah.... I'll make that change.